lib/autsr.gi

```diff
@@ -941,7 +941,11 @@ local d,a,map,possibly,cG,cH,nG,nH,i,j,sel,u,v,asAutomorphism,K,L,conj,e1,e2,
     u:=ClosureGroup(i,K);
```

Comment:
Typo in second commit message: `AmallGeneratingSet`

Reply:
fixed